net-im/gitter-bin/gitter-bin-3.1.0.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit eutils gnome2-utils unpacker
+
+DESCRIPTION="Where developers come to talk"
+HOMEPAGE="http://www.gitter.com/"
+SRC_URI="
+	amd64? ( https://update.gitter.im/linux64/${MY_PN}_${PV}_amd64.deb )
+	x86? ( https://update.gitter.im/linux32/${MY_PN}_${PV}_i386.deb )"
+LICENSE="no-source-code"
```

Comment:
I have not found any license in portage tree that can be used.
Did you mean that I have to submit a license for gitter?
If yes should this be done in another PR?